cloudmarker/events/azstorageaccountevent.py

```diff
@@ -0,0 +1,96 @@
+"""Microsoft storage account secure transfer event.
```

Comment:
The class name `AzStorageAccountEvent` does not capture the nature of this event plugin. A more apt name would be `AzStorageAccountSecureTransferEvent`.
There can be other events associated with an Azure storage account e.g. an event when default network rule is not set to deny